client/web/admin/src/views/System/Cdns/Index.vue

```diff
@@ -0,0 +1,257 @@
+            size="sm"
+            variant="danger"
+            @click="deleteCDN(modal.index)"
+          >
+            Delete
+          </b-button>
+          <b-button
+            size="sm"
+            variant="secondary"
+            @click="cancel()"
+          >
+            Cancel
+          </b-button>
+          <b-button
+            size="sm"
+            variant="primary"
+            @click="ok()"
+          >
+            OK
+          </b-button>
+        </template>
+      </b-modal>
+
+      <template #footer>
+        <c-button-submit
+          :disabled="!canManage"
+          :processing="processing"
+          :success="success"
+          :text="$t('admin:general.label.submit')"
+          class="ml-auto"
+          @submit="onSubmit()"
+        />
+      </template>
+    </b-card>
+  </b-container>
+</template>
+
+<script>
+import editorHelpers from 'corteza-webapp-admin/src/mixins/editorHelpers'
+import CdnProviderEditor from 'corteza-webapp-admin/src/components/Settings/System/CDNs/CSystemCdnProviderEditor'
+import { mapGetters } from 'vuex'
+
+export default {
+  name: 'CSystemCdnEditor',
+
+  i18nOptions: {
+    namespaces: 'system.cdns',
+    keyPrefix: 'editor',
+  },
+
+  components: {
+    CdnProviderEditor,
+  },
+
+  mixins: [
+    editorHelpers,
+  ],
+
+  data () {
+    return {
+      cdns: [],
+      modal: {
+        open: false,
+        editor: null,
+        title: null,
+        data: [],
+        index: null,
+      },
+
+      cdn: {
+        processing: false,
+        success: false,
+      },
+      originalCdns: [],
+    }
+  },
+
+  computed: {
+    ...mapGetters({
+      canManage: 'rbac/can',
+    }),
+
+    cdnProviderFields () {
+      return [
+        { key: 'provider', label: this.$t('cdns-provider.table-headers.provider'), thStyle: { width: '200px' }, tdClass: 'text-capitalize' },
+        { key: 'value', label: this.$t('cdns-provider.table-headers.value'), tdClass: 'td-content-overflow' },
+        { key: 'editor', label: '', thStyle: { width: '200px' }, tdClass: 'text-right' },
+      ]
+    },
+
+    providerItems () {
+      return this.cdns.map((s, i) => ({
+        provider: s.name,
+        value: s.cdnScript,
+
+        editor: {
+          component: 'cdn-provider-editor',
+          data: s,
+          index: i,
+          title: s.name,
+          updater: (changed) => {
+            this.cdns[i] = changed
+          },
+        },
+      }))
+    },
+  },
+
+  created () {
+    this.fetchSettings()
+
+    this.originalCdns = [...this.cdns]
+  },
+  methods: {
+    openEditor ({ component, title, data, updater }) {
+      this.modal.open = true
+      this.modal.component = component
+      this.modal.title = title
+      this.modal.updater = updater
+
+      // deref
+      this.modal.data = data
+    },
+
+    newCDN () {
+      this.openEditor({
+        component: 'cdn-provider-editor',
+        title: this.$t('cdns-provider.add'),
+        data: {},
```

Comment:
New CDNS should have `<script` as default value, since placeholders are a bit tricky here.